### ISSUE TITLE
Installation instructions for Opera, Safari, and Microsoft Edge?

### ISSUE THREAD

**Issue by sunjam** [damko/freedommarks-browser-webextension#6]
If it is briefly mentioned in the README, but are there Installation instructions for Opera, Safari, and Microsoft Edge?

**Comment by damko** [comment]
Hello @sunjam 
I've no clue if this extension can work on Opera, Safari or Microsoft Edge because I don't use them. Would you like be a beta tester for this?

**Comment by sunjam** [comment]
Absolutely, how would I install the app?  Let’s start with Safari.  Tips
appreciated.
